<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>248</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Video processing</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="topHorizontalLayout">
      <item>
       <widget class="QLabel" name="inputLabel">
        <property name="text">
         <string>Input video:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="inputPushButton">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="bottomHorizontalLayout">
      <item>
       <widget class="QLabel" name="outputLabel">
        <property name="text">
         <string>Output path:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="outputLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="outputPushButton">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="showButton">
      <property name="text">
       <string>Show video processing</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="processButton">
        <property name="text">
         <string>Start process</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
